src/components/StationsListItem.jsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 import styles from './StationsListItem.module.scss';
 import Button from './Button';
-import { ReactComponent as playLarge } from '../images/play-large.svg';
+import { ReactComponent as playMediumIcon } from '../images/play-medium.svg';
+import { toShorterUrl } from '../utils/shorter-url';
```

Comment:
Maybe a more verb-like name would be good here? Something like `shortenUrl`? (eng. shorten - to make sth shorter)